src/view/grant.njk

```diff
@@ -0,0 +1,107 @@
+              <li><b>Research</b>, e.g. economics PhD researching credit networks or P2P payment channel networks, especially with respect to the so-called “imbalanced channel” problem
+              <li><b>UX enhancements</b>, e.g. building a more advanced mobile application or integrating localisation features
+            </ul>
+
+            <h2>4. Our value proposition</h2>
+            <h3>Funding</h3>
+            <p>The total size of the grant depends on the scope of your project, your specific requirements and the estimated budget of the project. It will be paid out in TLN.</p>
+            <p>Furthermore, milestones might potentially be defined for each stage of the project during the application process. These milestones can include criteria that will be required to be met in order to release further payouts.</p>
+            <p>Lastly, further grants will also be available, subject to the progress of your project.</p>
+
+            <h3>Mentorship</h3>
+            <p>To ultimately leverage the  potential of your project, we also offer mentorship by providing you access to our network in the Ethereum ecosystem. For example, if you are looking for exchange opportunities, we are happy to use our connections to other community currency projects.</p>
+
+            <h3>Use of trademarks</h3>
+            <p>Last but not least, the Trustlines trademarks owned by the Foundation can be used by projects building on top of the Trustlines protocol in accordance with its Trustlines Trademark Guidelines. This way your project can already benefit from the popularity of the Trustlines protocol starting day one.</p>
+            <p>For example, this trademark will give you better access to the existing Trustlines community and thus increase your reach to promote your project to potential users.</p>
+            <p>Please find more information about our trademarks and how to work with them <a href="https://docs.trustlines.network/docs/foundation/trademarks.html" target="_blank">here</a>.</p>
+
+            <h2>5. How to apply</h2>
+            <p>As soon as  applications will open, we will be waiting for your submission. As of now, we are not planning to close the application window anytime soon. In short, you can apply anytime.</p>
+            <p>Once we receive your application, we will start the evaluation process. The evaluation process includes an initial screening of your application, followed by an in-depth review and potentially also an interview if necessary.</p>
+            <p>The duration of this process is highly dependent on the level of detail of your application and the scope of your project in general. Thus we ask for your understanding that we are not able to provide any specific timeframe in which the evaluation process will take place.</p>
+            <p>Please be aware that we might contact you  for additional information throughout the whole process in case we are missing any, so we kindly ask you to be available.</p>
+
```

Comment:
Add a line with " You can apply via Google Forms" and then hyperlink "Google Forms" to here: https://docs.google.com/forms/d/e/1FAIpQLSdC9g0QH1fLsNolH8saFwItYYtC0smFOVCBZPcRNUjITEbfQA/viewform?usp=sf_link